react/features/base/util/uri.ts

```diff
@@ -657,3 +659,21 @@ export function appendURLHashParam(url: string, name: string, value: string) {
 
     return newUrl.toString();
 }
+
+/**
+ * Sanitizes the given URL so that it's safe to use. If it's unsafe, null is returned.
+ *
+ * @param {string|URL} url - The URL that needs to be sanitized.
+ *
+ * @returns {URL?} - The sanitized URL, or null otherwise.
+ */
+export function sanitizeUrl(url: string | URL): URL | null {
+    const urlStr = url.toString();
```

Comment:
maybe check for url being undefined/null, otherwise it would prolly crash here.

Reply:
👍